peachjam/views/courts.py

```diff
@@ -125,40 +122,22 @@ def add_facets(self, context):
                 .distinct()
                 if attorney
             )
-            context["facet_data"]["attorneys"] = {
-                "label": _("Attorneys"),
-                "type": "checkbox",
-                "options": sorted([(a, a) for a in attorneys]),
-                "values": self.request.GET.getlist("attorneys"),
-            }
-
-        if "taxonomy" not in self.exclude_facets:
-            taxonomies = Taxonomy.objects.filter(
-                pk__in=self.form.filter_queryset(
-                    self.get_base_queryset(), exclude="taxonomies"
-                )
-                .filter(taxonomies__topic__isnull=False)
-                .order_by("taxonomies__topic__id")
-                .values_list("taxonomies__topic__id", flat=True)
-                .distinct()
-            )
+            if len(attorneys) > 1:
+                context["facet_data"]["attorneys"] = {
+                    "label": _("Attorneys"),
+                    "type": "checkbox",
+                    "options": sorted([(a, a) for a in attorneys]),
+                    "values": self.request.GET.getlist("attorneys"),
+                }
 
-            context["facet_data"]["taxonomies"] = {
-                "label": _("Topics"),
-                "type": "checkbox",
-                "options": sorted(
-                    [(t.slug, t.name) for t in taxonomies], key=lambda x: x[1]
-                ),
-                "values": self.request.GET.getlist("taxonomies"),
-            }
-
-        if "alphabet" not in self.exclude_facets:
-            context["facet_data"]["alphabet"] = {
-                "label": _("Alphabet"),
-                "type": "radio",
-                "options": [(a, a) for a in lowercase_alphabet()],
-                "values": self.request.GET.get("alphabet"),
-            }
+    def add_facets(self, context):
+        context["facet_data"] = {}
+        self.get_judges_facet(context)
```

Comment:
Minor: Use `add_...` rather than `get_...` because `get` usually means that there is no side-effect and the method returns a result.